general/tools.md

```diff
@@ -3,20 +3,26 @@
 At source{d} we try to use tools that are as simple as possible. If you can think of any tools that are simpler, better or cheaper please let us know! Here we will give instructions on how to make the best use of the tools we currently use.
 
 ## General
+
 These are the tools that every sourcerer uses:
 * [Github](https://github.com)
 * [Slack](https://slack.com)
-* [Gmail](https://gmail.com)
-  * There is no internal email at source{d}, we use GitHub for all async communications
-* [Zoom](http://zoom.us/)
-* [Hangouts Meet](http://hangouts.google.com)
+* [Gmail](https://gmail.com) (internal email usage is minimal at source{d}, most communication happens on Slack or GitHub)
+* [KinHR](https://kinhr.com)
```

Comment:
It might be worth including a brief summary of what KinHR is; I think everyone knows what GitHub, Slack, and GMail are, but I'd never heard of KinHR before I got here. (Though maybe that's just me)

Reply:
```suggestion
* [KinHR](https://kinhr.com), to manage your days off, payslips, copies of your contract, social security and such.
```